website/content/specification/syntax/constraints.md

```diff
@@ -211,12 +211,11 @@ The [`id`](#id) attribute of an `is-unique` constraint specifies an identifier f
 
 The `@target` attribute of an `<is-unique>` constraint identifies the node(s) in a document instance that are checked for uniqueness based on the computed key. The `is-unique` constraint MUST define a [`@target`](#target) with a Metapath expression.
 
-The `key-field` field of an `<is-unique>` constraint defines the attribute or element value that is the key for each entry in the index. The `<key-field>` element MUST define a [`@target`](#target) attribute with a Metapath expression evaluated relative to [the evaluation focus](#constraint-processing) of each entry of the matches of the constraint's `@target`.
+A `<key-field>` field of an `<is-unique>` constraint defines the attribute or element value that is the key for each entry in the index. A `<key-field>` element MUST define at least one [`@target`](#target) attribute with a Metapath expression evaluated relative to [the evaluation focus](#constraint-processing) of each entry of the matches of the constraint's `@target`.
```

Comment:
Close, but "at least one" target sounds like target is optional on key-field when it is mandatory.

```suggestion
A `<key-field>` assembly of an `<is-unique>` constraint defines the flag or field value that is the key for each entry in the index. The `<key-field>` field MUST define a [`@target`](#target) flag with a Metapath expression evaluated relative to [the evaluation focus](#constraint-processing) of each entry of the matches of the constraint's `@target`.
```